Packs/CommonWidgets/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,27 @@
+
+#### Scripts
+##### GetLargestInputsAndOuputsInIncidents
+- Removed the MB suffix from the table header **Size**.
+- Changed table header name **Size**->**Size(MB)**.
+- Changed table header **IncidentID** to a hyperlink to the incident.
+- Changed table header **TaskID** to a hyperlink to the task.
+- Changed script result format to text markdown format, rather than table format.
+##### GetLargestInvestigations
+- Removed the MB suffix from the table header **Size**.
+- Changed table header name **Size**->**Size(MB)**.
+- Changed table header **IncidentID** to a hyperlink to the incident.
+- Changed default script result format to text markdown format, rather than table format.
+- Added handling for **playground** investigation.
+- Added new argument **table_result** return a result suitable for a table widget. By default
+    the script result will be in Markdown.
+
+#### Widgets
+##### Largest Incidents by Storage Size
+- Removed the MB suffix from the table header **Size**.
+- Changed table header name **Size**->**Size(MB)**.
+- Changed table header **IncidentID** to a hyperlink to the incident.
+##### Largest Inputs And Outputs In Incidents
+- Removed the MB suffix from the table header **Size**.
+- Changed table header name **Size**->**Size(MB)**.
+- Changed table header **IncidentID** to a hyperlink to the incident.
+- Changed table header **TaskID** to a hyperlink to the task.
```

Comment:
maybe we should make it less verbose. @kirbles19 WDYT?

Reply:
I'll work on these.